src/DataSet/DataSetColumn.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of itk-dev/datatidy.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\DataSet;
+
+use Doctrine\DBAL\Schema\Column;
+use Doctrine\DBAL\Types\Type;
+use InvalidArgumentException;
+
+/**
+ * A data set column with both a display name and a sql name.
+ *
+ * It basically acts as a shim around an ORM schema column.
+ */
+class DataSetColumn extends Column
+{
+    public function __construct($columnName, Type $type, array $options = [])
+    {
+        parent::__construct($columnName, $type, $options);
+        $this->getSqlName();
+    }
+
+    public static function create(Column $column): self
+    {
+        $options = $column->toArray();
+        unset($options['name'], $options['type']);
+
+        return new self($column->_name, $column->_type, $options);
+    }
+
+    /** @var string */
+    private $sqlName;
```

Comment:
Maybe this should be moved to top of class?

Reply:
Moved.